sfa_dash/templates/forms/obs_data_download_form.html

```diff
@@ -0,0 +1,66 @@
+{% extends "dash/data.html" %}
+{# {% set sidebar = True %} #}
+{% set page_title = 'Download Observation data' %}
+{% block content %}
+{{ metadata | safe }}
+<h3>Download data for Observation 'GHI Instrument 1'</h3>
+{# <p><b>Available Date Range:</b> 2016-01-01T12:00Z to 2019-01-01T12:00Z</p> #}
+<form action="{{ url_for('forms.download_observation_data', uuid=uuid) }}" method="post" id="observation-download">
+  <div class="form-group">
+	<div class="form-element">
+	  <label for="start">Start time:</label>
+	  <input type="datetime-local" id="start" name="period-start">
+    </div>
+    <div class="form-element">
+	  <label for="end">End time:</label>
+	  <input type="datetime-local" id="end" name="period-end">
+    </div>
+<div class="form-element">
+      <label>Timezone</label>
+      <select required name="timezone" class="form-control">
+        <option value="utc">UTC</option>
+      </select>
+    </div>
+    <div class="form-element full-width">
+      <label for="format">Format</label><br/>
+      <input type="radio" data-toggle="collapse" data-target=".download-example" name="format" value="text/csv" checked>CSV</input>
+      <input type="radio" data-toggle="collapse" data-target=".download-example" name="format" value="application/json">JSON</input>
+    </div>
+    <div class="form-element full-width">
+      <div class="download-example csv show">
+      <p>CSV downloads will have the following format.</p>
+      <pre class="example-text">
+# name: GHI Instrument 1
+# obs_id: 123e4567-e89b-12d3-a456-426655440000
```

Comment:
definitely not `obs_id`. observations_id? id?

Reply:
I agree. I think this should be a sweeping change though, maybe made in it's own PR to each of our repos.